test/vm.install

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -eux
+
+SRPM="$1"
+
+# Grow root partition to make room for images. This only works on Fedora right now.
+echo ", +" | sfdisk -N 2 -f /dev/vda
+partprobe
+pvresize /dev/vda2
```

Comment:
This works on RHEL images, too, as "vda2" is defined by the outside image build process and QEMU/libvirt invocation. If you want to keep it a bit more flexible, use 

    pvs --noheadings -opv_name | xargs pvresize